office/libreoffice/Portfile

```diff
@@ -77,7 +78,7 @@ depends_build-append \
 depends_lib-append  \
     port:apr \
     port:apr-util \
-    port:boost \
+    port:[boost::depends_portname] \
```

Comment:
with the `boost` PG you shouldn't include the port manually anymore (see also the `port lint` warning)